### PR TITLE
Text changes

### PR DESCRIPTION
**Pull request by audax**

### CODE REVIEW

File: qabelbox/build.gradle [hunk 1]
```diff
@@ -29,7 +29,7 @@ android {
         add code to automaticly generate version name and code,
         on base of timestamp or change it manual*/
 
-        versionCode System.getenv("BUILD_NUMBER") as Integer ?: 1
+        versionCode System.getenv("BUILD_NUMBER") as Integer ?: 3
         versionName "0.2." + (System.getenv("BUILD_NUMBER") as Integer ?: 3)
```
Comment (thechauffeur) [inline qabelbox/build.gradle]: According to the version scheme currently in use now this should result in `0.2.4`.

File: qabelbox/src/main/res/values-de/strings.xml [hunk 2]
```diff
@@ -209,12 +209,12 @@
...
     <string name="message_welcome_screen2">Ende-zu-Ende verschlüsselter\nUpload, Download und\nSpeicherung.\nMetadatenverschleierung.\n2 GB kostenloser Speicher.\nSynchronisation über beliebig\nviele Endgeräte.\nSharing mit anderen\nQabelnutzern.\nVerwaltung mehrerer Identitäten\nmit nur einem Qabel-Account\nund in nur einer Anmeldung.\nIntegrierter\nKurznachrichtendienst.</string>
-    <string name="message_welcome_screen3">Qabel erlaubt es Ihnen,\nmehrere Identitäten in nur\neinem Account abzubilden.\nSie können verschiedene\nIdentitäten auf\nverschiedenen Endgeräten\nsynchronisieren und\nbestimmen, welche\nIdentitäten Sie mit anderen\nMenschen Teilen möchten.\nSeinen Sie Bob für\nIhren Chef und Alice für\nIhre Freunde!</string>
+    <string name="message_welcome_screen3">Qabel erlaubt es Ihnen,\nmehrere Identitäten in nur\neinem Account abzubilden.\nSie können verschiedene\nIdentitäten auf\nverschiedenen Endgeräten\nsynchronisieren und\nbestimmen, welche\nIdentitäten Sie mit anderen\nMenschen Teilen möchten.\nSenen Sie Bob für\nIhren Chef und Alice für\nIhre Freunde!</string>
```
Comment (thechauffeur) [inline qabelbox/src/main/res/values-de/strings.xml]: Not `Senen` but `Seien`.

Comment (jasinai) [inline qabelbox/src/main/res/values-de/strings.xml]: And `teilen`

**Comment by thechauffeur** [comment]
English version for `Danksagung` is missing

File: qabelbox/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -209,9 +209,57 @@
 
     <string name="licence.header.intro">Qabel is Published under the QAPL:</string>
     <string name="about.header.qabel.company.qtxt">QABEL</string>
-    <string name="about.header.qabel.show.btn">Show QAPL</string>
+    <string name="about.header.qabel.show.btn">Show QaPL</string>
     <string name="about.header.qabel.thanks.header.txt"> bedankt sich bei:</string>
-    <string name="about.header.qabel.thanks.content.txt">praemandatum GmbH\nden Unterstützern unserer Indiegogo Kampagne\nHinz\nund\nKunz</string>
+    <string name="about.header.qabel.thanks.content.txt">Alexander Hartmann
```
Comment (thechauffeur) [inline qabelbox/src/main/res/values/strings.xml]: This text must begin with `Qabel bedankt sich bei:` plus two newlines.

KEEP IN MIND that the text `Qabel` at the beginning is shown as the Qabel logo.

File: qabelbox/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -209,9 +209,57 @@
 
     <string name="licence.header.intro">Qabel is Published under the QAPL:</string>
     <string name="about.header.qabel.company.qtxt">QABEL</string>
-    <string name="about.header.qabel.show.btn">Show QAPL</string>
-    <string name="about.header.qabel.thanks.header.txt"> bedankt sich bei:</string>
-    <string name="about.header.qabel.thanks.content.txt">praemandatum GmbH\nden Unterstützern unserer Indiegogo Kampagne\nHinz\nund\nKunz</string>
+    <string name="about.header.qabel.show.btn">Show QaPL</string>
+    <string name="about.header.qabel.thanks.header.txt"> thanks:</string>
```
Comment (thechauffeur) [inline qabelbox/src/main/res/values/strings.xml]: I can't find this header for the german version. Is it included?

**Pull request merged**